app/code/Magento/Sales/Model/Order.php

```diff
@@ -214,6 +216,11 @@ class Order extends AbstractModel implements EntityInterface, OrderInterface
     protected $_currencyFactory;
 
     /**
+     * @var \Magento\Eav\Model\Config
+     */
+    protected $_eavConfig;
```

Comment:
I would give it a try and if no core tests are broken leave it `private`.

I believe if any extension relied on this property as public it's totally weird, inheritance from core classes is not recommended and property is not marked as `@api`.